From 116e6085b5ea09124b14754b24e643b1bdf1a3ce Mon Sep 17 00:00:00 2001
From: Wang Quanyang <quanyang.wang@windriver.com>
Date: Tue, 13 Oct 2015 08:29:48 -0400
Subject: [PATCH 1091/1094] mmc: esdhc: replace adma_desc with adma_table

According to upstream commit 4efaa6fbe1fd
("mmc: sdhci: Rename adma_desc to adma_table"), use
adma_table instead of adma_desc.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mmc/host/sdhci-of-esdhc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/sdhci-of-esdhc.c b/drivers/mmc/host/sdhci-of-esdhc.c
index d65f44f..0137e70 100644
--- a/drivers/mmc/host/sdhci-of-esdhc.c
+++ b/drivers/mmc/host/sdhci-of-esdhc.c
@@ -284,7 +284,7 @@ static void esdhci_of_adma_workaround(struct sdhci_host *host, u32 intmask)
 			host->data->error = 0;
 
 		BUG_ON(!host->data);
-		desc = host->adma_desc;
+		desc = host->adma_table;
 		for_each_sg(host->data->sg, sg, host->sg_count, i) {
 			addr = sg_dma_address(sg);
 			offset = (4 - (addr & 0x3)) & 0x3;
@@ -298,7 +298,7 @@ static void esdhci_of_adma_workaround(struct sdhci_host *host, u32 intmask)
 		 * terminating descriptor.
 		 */
 		desc += 8;
-		WARN_ON((desc - host->adma_desc) > (128 * 2 + 1) * 4);
+		WARN_ON((desc - host->adma_table) > (128 * 2 + 1) * 4);
 
 		dataddr = (__le32 __force *)(desc + 4);
 		cmdlen = (__le32 __force *)desc;
-- 
2.0.2

